dev/benchmark/sysbench-v4.md

```diff
@@ -0,0 +1,252 @@
+---
```

Comment:
This file was updated and renamed to `how-to-run-sysbench.md` in https://github.com/pingcap/docs/pull/1213.